flake8_black.py

```diff
@@ -98,14 +104,11 @@ def _file_mode(self):
                 skip_string_normalization = black_config.get(
                     "skip_string_normalization", False
                 )
-
-            if self.line_length != black_line_length:
-                raise ValueError(
-                    "conflict in line-length param - "
-                    "flake8 use {} and black use {}".format(
-                        self.line_length, black_line_length
+                if self.line_length != black_line_length:
```

Comment:
Hmm, actually, correct predicate for checking configuration conflict is interesting moment.
Obviously, situation, when _black_line_length > line_length_ is incorrect, because formatted code cannot pass flake8 check for line-lenght.
Opposite situation when _black_line_length < line_length_ is more interesting. Formally speaking, this situation is correct, because formatted code must pass flake8 check for line-lengh. But it actually means, that formatting rules is too strong according to flake8 rules and this mismatch can be "unpleasant" for the user.

In my opinion, there is should be **only one value** for line-length for both flake and black and we should nofify about any difference in line-length value.

@peterjc what do you think about this situation?